docs/content/querying/dimensionspecs.md

```diff
@@ -396,6 +396,10 @@ or without setting "locale" (in this case, the current value of the default loca
 
 ### Lookup DimensionSpecs
 
+<div class="note caution">
+Lookups are an experimental feature.
```

Comment:
some details about what this means would be nice.

do either of these apply?

- api may change in backwards incompatible ways
- 'beta' - not yet validated across a variety of production clusters